backend/business-partner-agent/src/main/resources/bcgov-ux.yml

```diff
@@ -0,0 +1,56 @@
+bpa:
```

Comment:
I think it's not a good practice to add config files that are specific to an organization to the resources folder. Instead I would propose to reference or generate this in the helm chart and then overwrite the settings in application.yml with `-Dmicronaut.config.files=classpath:application.yml,/home/indy/bcgov-ux.yml`

Reply:
Correct. That was from when I started out. I have removed that file. I am using the bcgov overrides as a commented out example so new developers can see what is possible/allowed.